pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -704,20 +705,20 @@ func (m *kubeGenericRuntimeManager) doPodResizeAction(pod *v1.Pod, podContainerC
 	podResources := cm.ResourceConfigForPod(pod, enforceCPULimits, uint64((m.cpuCFSQuotaPeriod.Duration)/time.Microsecond), false)
 	if podResources == nil {
 		klog.ErrorS(nil, "Unable to get resource configuration", "pod", klog.KObj(pod))
-		result.Fail(fmt.Errorf("unable to get resource configuration processing resize for pod %s", pod.Name))
-		return
+		resizeResult.Fail(kubecontainer.ErrResizePod, fmt.Sprintf("unable to get resource configuration processing resize for pod %s", pod.Name))
```

Comment:
If you use the generic `ErrResizePod` here, that might be all that gets logged when the error is processed. We want to make sure the full error description is logged, so I think it's better to just use the same string in both cases:

```suggestion
		msg := fmt.Sprintf("unable to get resource configuration processing resize for pod %s", pod.Name)
		resizeResult.Fail(errors.New(msg), msg)
```

Reply:
(same everywhere else this is used)

Reply:
fwiw, this is what I'm seeing in my local run (both the error and message getting logged, which makes sense because of the way [the error is formatted here](https://github.com/kubernetes/kubernetes/blob/1039822a7432bdf822b6dc32f20100499ec7d22e/pkg/kubelet/container/sync_result.go#L122)):

```
Mar 19 17:27:27 kind-worker kubelet[235]: E0319 17:27:27.524842     235 pod_workers.go:1301] "Error syncing pod, skipping" err="failed to \"ResizePod\" for \"test-ns_test-pod(e4c54db8-fe50-43d4-92dd-9d1e1a64bf0d)\" with ResizePodError: \"unable to get resource configuration processing resize for pod test-ns\"" pod="test-pod/test-ns" podUID="e4c54db8-fe50-43d4-92dd-9d1e1a64bf0d"
```

I think if we update this to make `Error` the same as `Message`, it'll look like this:

```
Mar 19 17:27:27 kind-worker kubelet[235]: E0319 17:27:27.524842     235 pod_workers.go:1301] "Error syncing pod, skipping" err="failed to \"ResizePod\" for \"test-ns_test-pod(e4c54db8-fe50-43d4-92dd-9d1e1a64bf0d)\" with unable to get resource configuration processing resize for pod test-ns: \"unable to get resource configuration processing resize for pod test-ns\"" pod="test-pod/test-ns" podUID="e4c54db8-fe50-43d4-92dd-9d1e1a64bf0d"
```

Idk if you are thinking of another place where it's getting logged though. If you still want me to make this update I'll defer to your preference.

Reply:
Ah, ok. That's a little weird, but what you have makes sense in this context. Let's leave it as is then.